ZelBack/src/services/appsService.js

```diff
@@ -3364,11 +3366,10 @@ async function registerAppLocally(appSpecs, componentSpecs, res) {
     }
 
     if (!isComponent) {
-      // last character of appName determines gateway
-      const lastCharCode = appName.charCodeAt(appName.length - 1);
-
-      const dockerNetworkAddrValue = lastCharCode;
-
+      let dockerNetworkAddrValue = Math.floor(Math.random() * (254 - 0 + 1) + 0);
```

Comment:
why -0 +1 + 0?

Reply:
0-255?
Math. floor(Math. random() * 256)

Reply:
Math.floor(Math.random() * 255);

Reply:
fixed

Reply:
so 254 is maximum then? not 255? 255 is still valid no?

Reply:
fixed